src/core/python2js.c

```diff
@@ -260,6 +260,13 @@ _python2js_immutable(PyObject* x)
     return _python2js_long(x);
   } else if (PyFloat_Check(x)) {
     return _python2js_float(x);
+  } else if (PyNumber_Check(x)) {
```

Comment:
I think we need a regression test for at least one type included in `PyNumber_Check` https://github.com/python/cpython/blob/f0a6fde8827d5d4f7a1c741ab1a8b206b66ffd57/Objects/abstract.c#L756 but not selected in earlier `else if` branches.

Reply:
I think the right approach is to add a test with a class with an `__int__` method that returns a constant and one that throws an error.

Reply:
Actually I think this change is misguided so I'll just remove it.